app/src/main/java/com/github/dedis/student20_pop/PoPApplication.java

```diff
@@ -24,12 +24,17 @@
  * Class modelling the application : a unique person associated with LAOs
  */
 public class PoPApplication extends Application {
+    public static final String TAG = PoPApplication.class.getSimpleName();
 
     private static final String LOCAL_BACKEND_URI = "ws://10.0.2.2:2000";
+    public static final int ADD_WITNESS_SUCCESSFUL = 0;
+    public static final int ADD_WITNESS_ALREADY_EXISTS = 1;
```

Comment:
I would rather have a boolean return value or a dedicated enum with the possible return values.

I don't really like int return values in java

Reply:
Done :+1: